### ISSUE TITLE
pre-trained weights from coco dataset

### ISSUE THREAD

**Issue by coldgemini** [facebookresearch/Detectron#51]
hello, but do you also have pretrained weights from coco dataset instead of imagenet?

thanks

**Comment by ppwwyyxx** [comment]
[Proposal, Box, and Mask Detection Baselines](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#proposal-box-and-mask-detection-baselines)

These models are all trained on coco.

**Comment by coldgemini** [comment]
yes, but my dataset has class number different from coco dataset so your mentioned model cannot be directly loaded into the training scripts. I don't know how caffe2 handles model importing with different layer sizes. I know however in caffe I can just modify the layer name.

**Comment by KaimingHe** [comment]
To further fine-tune a COCO model on another dataset (e.g., Cityscapes as we did), you need to remove the layers that are class-dependent (in Mask R-CNN, in bbox cls, bbox reg, and mask pred layers), and reinitialize them.

In Detectron, the weights are loaded by matching names, so a simple way to do this is either to change the name of the class-dependent layers when you do further fine-tuning, or add a flag in the weight loading to not load these weights.
